charts/scalardb/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
```

Comment:
Is there a reason why do not have the following at the beginning such as scalardl?
```
{{/* vim: set filetype=mustache: */}}
```

Reply:
I left this as default generated by helm so it doesn't have. I'll add the above line.